### ISSUE TITLE
Skin appears monochromatic

### ISSUE THREAD

**Issue by BloveDawn** [pmmp/PocketMine-MP#3278]
### Issue description

- Expected result: What were you expecting to happen?
Normal skin loading
- Actual result: What actually happened?
Abnormal skin loading

### Steps to reproduce the issue
1. Set up the server normally without any plug-ins
2. Enter the game, the skin is shown as monochrome

![image](https://user-images.githubusercontent.com/40706446/72577459-ce7edc80-390d-11ea-815c-037ed3b4273f.png)

### OS and versions

* PocketMine-MP: 3.11.3
* PHP: 7.3
* Server OS: Ubuntu18.04
* Game version: PE&Win10

`This server is running PocketMine-MP 3.11.3 for Minecraft: Bedrock Edition v1.14.0 (protocol version 389)`

### Plugins

NO PLUGINS

### Crashdump, backtrace or other files

NO CRASHDUMP FILE

**Comment by Wumpotamus** [comment]
The newer skins (persona) from 1.13 and up aren't supported

**Comment by jasonwynn10** [comment]
This is due to Mojang's change in skins. The new 'persona' type skins don't have full compatibility on servers, so they are replaced with monochromatic ones instead. A similar bug can be found in Mojang's official software also.

TLDR: this is intentional to prevent you from crashing your client

**Comment by dktapps** [comment]
https://github.com/pmmp/PocketMine-MP/blob/stable/changelogs/3.10.md#note-about-skins
